coordinator/src/services/translation/xml.ts

```diff
@@ -1,10 +1,19 @@
 import * as XmlJs from "xml-js";
 
 export function writeXmlStringCanonicalized(tag: XmlJs.ElementCompact): string {
+    return writeXml(tag, 0, true)
+}
+
+export function writeXmlStringPretty(tag: XmlJs.ElementCompact): string {
+    return writeXml(tag, 2, false)
+}
+
+function writeXml(tag: XmlJs.ElementCompact, spaces: number, fullTagEmptyElement: boolean): string {
     const options = {
         compact: true,
+        spaces,
```

Comment:
Do these not need keys?

Reply:
No, if the field name and key are the same, you can just specify ti once